drivers/SmartThings/zigbee-window-treatment/src/hanssem/init.lua

```diff
@@ -0,0 +1,259 @@
+-- JJSmart Window Shade ver 0.1
```

Comment:
this name here is confusing. What's JJSmart?

Reply:
It has been deleted.